src/web/timeline.rkt

```diff
@@ -270,6 +271,16 @@
               `(tr (td ,(~a count) "×")
                    (td (code ,(~a rule) " "))))))))
 
+(define (render-phase-problems problems)
+  `((dt "Problems")
+    (dd (table ([class "times"])
+          ,@(for/list ([rec (in-list (sort problems > #:key second))])
+              (match-define (list expr count) rec)
+              `(tr (td ,(~a count) "×")
+                   (td ,(if (equal? expr #f)
```

Comment:
You can just do `if expr` and flip the branches, `#f` is the only false thing in Racket.